src/main/java/io/prometheus/cloudwatch/HealthServlet.java

```diff
@@ -0,0 +1,15 @@
+package io.prometheus.cloudwatch;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class HealthServlet extends HttpServlet {
+
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        resp.setContentType("text/html");
+        resp.getWriter().print("OK");
```

Comment:
This isn't valid HTML.

Reply:
Fair, i took inspiration from the prometheus health endpoint, which I don't believe is valid html either... would:

```html
<html>ok</html>
```

be sufficient?

Reply:
Switching to text/plain is simplest.

Reply:
I guess to be html valid it'd have to be:
```html
<!doctype html>
<html lang=en>
<head>
<meta charset=utf-8>
<title>cloudwatch exporter</title>
</head>
<body>ok
</body>
</html>
```

Reply:
Thanks, i'm not a smart man :)